Earthfile

```diff
@@ -1,4 +1,4 @@
-VERSION 0.7
+VERSION 0.6
```

Comment:
This shouldn't downgrade.

Reply:
I wasn't able to get it to work on onrender.com, I'll try 0.7 again.

Reply:
GitHub CI still doesn't push the image to ghcr when on Earthly v0.7. v0.6 still works fine.

Reply:
What do you mean by "doesn't push", is there an error?

Reply:
v0.7 (doesn't push to ghcr)

https://github.com/elliotwutingfeng/tokei_rs/actions/runs/5494842291/jobs/10013869341

Under `Run build` step

```code
              context | --> local context .
              output | --> exporting outputs

 Push Summary ⏫
————————————————————————————————————————————————————————————————————————————————

Did not push image /cache/build
Did not push image /cache/deps
Pushed image github.com/elliotwutingfeng/tokei_rs:dev+docker as 
Did not push image /cache/install-chef
Did not push image /cache/prepare

 Local Output Summary 🎁 (disabled)
```


v0.6 (still works)

https://github.com/elliotwutingfeng/tokei_rs/actions/runs/5494972413/jobs/10014066312

Under `Run build` step

```code
              context | --> local context .
              output | --> exporting outputs
              output | [          ]   0% pushing layers
              output | [----------] 100% pushing layers
              output | [          ]   0% pushing manifest for ghcr.io/elliotwutingfeng/tokei_rs:latest@sha256:1341c938af74d8074829ba25289e1bc77b90b9f41f208d7ee8eaa76ef14ee892
              output | [----------] 100% pushing manifest for ghcr.io/elliotwutingfeng/tokei_rs:latest@sha256:1341c938af74d8074829ba25289e1bc77b90b9f41f208d7ee8eaa76ef14ee892

 Push Summary ⏫
————————————————————————————————————————————————————————————————————————————————


 Local Output Summary 🎁 (disabled)
```

Reply:
Ah I think we just need to add `--push` to either the CI script or earthfile

Reply:
The CI script already has `--push` i.e. `earthly --ci --push +docker --repo 'ghcr.io/elliotwutingfeng/tokei_rs'`

Reply:
I got the `--push` working by placing the **ARG** statements within each target, it appears to be a scope-related bug similar to https://github.com/earthly/earthly/issues/2887

https://github.com/elliotwutingfeng/tokei_rs/actions/runs/5500568688/jobs/10023563976